frontend/src/App.js

```diff
@@ -1,10 +1,22 @@
-import logo from './logo.svg';
-import './App.css';
+import './App.css'
+import React from "react";
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import SearchAddress from './components/SearchAddress';
+import 'bootstrap/dist/css/bootstrap.css';
+
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
+      <Router>
+        <Routes>
+          {/* <Route exact path ="/" element={<Home/>} /> */}
+
+          <Route path ="/search" element={<SearchAddress/>} />
```

Comment:
Just fyi, if you want to put more elements inside the element prop of the route you can pass it as a child as well e.g.:
`<Route path="/search">
   <SearchAddress />
   <Route />`
   
You could think about passing always a file with the page suffix (e.g. <SearchPage />) which then contains all Components that should show on that specific page. It helps to keep the file structure organised and provides a better understanding what happens here for other devs.

(Again a point that applies if we talk about scalability rather than this specific use case)